Language/Haskell/GhcMod/PathsAndFiles.hs

```diff
@@ -114,3 +114,13 @@ setupConfigPath = localBuildInfoFile defaultDistPref
 
 packageCache :: String
 packageCache = "package.cache"
+
+-- | @findCradleFile dir@. Searches for a @.ghc-mod.cradle@ file in @dir@.
+-- If it exists in the given directory it is returned otherwise @findCradleFile@ returns @Nothing@
+findCradleFile :: FilePath -> IO (Maybe FilePath)
+findCradleFile directory = do
+    let path = directory </> ".ghc-mod.cradle"
```

Comment:
I'd rather this wasn't a hidden file, might make it hard to know what is happening for users.